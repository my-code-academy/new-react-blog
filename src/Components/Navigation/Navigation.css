.nav {
    padding: 10px 80px;
    background: #000;
    color: #fff;
    display: flex;
    justify-content: space-between;
    align-items: flex-end;
    height: 60px;
  }


  .nav div h2{
    margin-top:-20px;
    font-style: italic;
    font-size:1em;
  }
  .navLink a {

    color:white;
    text-decoration: none;
  }

  .navLink a:nth-child(2){
      margin:0 100px;
  }