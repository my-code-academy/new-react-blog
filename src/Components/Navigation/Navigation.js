import React, { Component } from 'react';
import './Navigation.css';

class Navigation extends Component {
  render() {
    return (
      <div className="nav">
      <div><h1>The Artifact</h1>
       <h2>Culture & Art Blog</h2></div>
       <div className="navLink">
        <a href="#!">Blog</a>
       <a href="#!">About</a>
       <a href="#!">Contact</a>
       </div>
      </div>
    );
  }
}

export default Navigation;
