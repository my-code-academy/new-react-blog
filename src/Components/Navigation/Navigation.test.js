import React from 'react';
import renderer from 'react-test-renderer';
import Navigation from './Navigation';

describe('Navigation', () => {
  it("should render a Navigation bar", () => {
    const tree = renderer.create(<Navigation/>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});