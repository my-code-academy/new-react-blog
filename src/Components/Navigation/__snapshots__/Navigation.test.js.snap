// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation should render a Navigation bar 1`] = `
<div
  className="nav"
>
  <div>
    <h1>
      The Artifact
    </h1>
    <h2>
      Culture & Art Blog
    </h2>
  </div>
  <div
    className="navLink"
  >
    <a
      href="#!"
    >
      Blog
    </a>
    <a
      href="#!"
    >
      About
    </a>
    <a
      href="#!"
    >
      Contact
    </a>
  </div>
</div>
`;
